### PR TITLE
Allow manual (force) updates to eggs

### LINKED ISSUE #310: [Eggs] - Allow force updating of a single egg with any json file
Allow users to update a single egg on the view egg page disregard if the uuids match or not, with confirmation that the user wants to overwrite the current egg with the newly uploaded egg.

### PR DESCRIPTION
**Pull request by lancepioch**
Resolves #310

### CODE REVIEW

File: app/Filament/Resources/EggResource/Pages/EditEgg.php [hunk 1]
```diff
@@ -200,15 +205,91 @@ protected function getHeaderActions(): array
...
+                ])
+                ->action(function (array $data, Egg $egg): void {
+                    /** @var EggImporterService $eggImportService */
+                    $eggImportService = resolve(EggImporterService::class);
+
+                    if (!empty($data['egg'])) {
+                        /** @var TemporaryUploadedFile[] $eggFile */
+                        $eggFile = $data['egg'];
+
+                        foreach ($eggFile as $file) {
```
Comment (RMartinOscar) [inline app/Filament/Resources/EggResource/Pages/EditEgg.php:256]:
```suggestion
                        foreach ($eggFile as $file) {
                            try {
                                $eggImportService->fromFile($file, $egg);
                            } catch (Exception $exception) {
                                Notification::make()
                                    ->title('Import Failed')
                                    ->danger()
                                    ->send();

                                report($exception);

                                return;
                            }
                        }
```
```suggestion
                        try {
                            $eggImportService->fromFile($file, $egg);
                        } catch (Exception $exception) {
                            Notification::make()
                                ->title('Import Failed')
                                ->danger()
                                ->send();

                            report($exception);

                            return;
                        }
```

**Review by RMartinOscar — COMMENTED** [review]
Don't need the foreach since we are not using ->multiple()

**Review by Boy132 — CHANGES_REQUESTED** [review]
~~This would still check on the UUID but the force update should ignore the UUID.~~

**Review by Boy132 — APPROVED** [review]
LGTM

**Review by notAreYouScared — APPROVED** [review]

**Pull request merged**